Library/Subscription.cs

```diff
@@ -374,6 +374,16 @@ public bool UpdateNotes(Dictionary<string, string> notes)
             return true;
         }
 
+        public RecurlyList<CouponRedemption> GetRedemptions()
```

Comment:
I wonder if we need to change this method name to `GetActiveRedemptions` like the account object